### DISCUSSION TITLE
Recording Board meetings where practical

### DISCUSSION THREAD

**Discussion by glennawatson** [dotnet-foundation/Home#57] [General Discussions]
I suggest that maybe board meetings (particular when there are votes) should be recorded.

I know from talking with previous board members that sometimes huge discussions items will become a single bullet point on a meeting minutes and looses a lot of the context and meaning.

There might be times when you can't record, such as dealing with confidential matters like members/board members personal lives, disciplinary actions, needing a safe place to talk, but in general, most formal meetings should be recorded and placed somewhere members have access.

**Comment by gortok** [comment]
Historically this is [what the minutes of the meeting](http://www.rulesonline.com/rror-10.htm#60) are for. According to the existing by-laws there should be minutes of all meetings by the Board of directors (Which would include the 'daily' meetings you all had last week, preceding and following the resignation of the Executive Director).  If that's not happening, it should be.

Article VI, Section 3 (e):

>  Minutes of the proceedings of the Board of Directors, and any minutes that may be maintained by committees having any of the authority of the Board of Directors;

Note, the lowercase `proceedings` used here, as opposed to the capitalized 'Proceedings' which has a different meaning in the by-laws. 

Now, all that said; if Board meetings were 'live', then members could just watch the proceedings.  That would, combined with the minutes, allow members to have a pretty good idea of what's going on -- even when there is a closed session.
